Description: specs: Fix expected hash function (SHA256 vs SHA512)
 The upstream specs expect the hash function to be SHA256, whereas
 Debian uses SHA512.
Author: Georg Faerber <georg@riseup.net>
Forwarded: not yet
Last-Update: 2018-05-26
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/spec/schleuder/integration/keywords_spec.rb
+++ b/spec/schleuder/integration/keywords_spec.rb
@@ -1602,7 +1602,7 @@
     raw = Mail::TestMailer.deliveries.first
     message = Mail.create_message_to_list(raw.to_s, list.email, list).setup
 
-    expect(message.to_s.gsub("\r", '')).to match(/BEGIN PGP SIGNED MESSAGE-----\nHash: SHA256\n\n#{signed_text}-----BEGIN PGP SIGNATURE/)
+    expect(message.to_s.gsub("\r", '')).to match(/BEGIN PGP SIGNED MESSAGE-----\nHash: SHA512\n\n#{signed_text}-----BEGIN PGP SIGNATURE/)
 
     teardown_list_and_mailer(list)
   end
